README.md

```diff
@@ -62,6 +62,22 @@ The result looks like this:
 
 If Mercury is unable to find a field, that field will return `null`.
 
+## `parse()` Options
```

Comment:
Let's make this and the `The command line parser` section below h4s, so `####` (That way they're still hierarchically under `Usage`.